docs/v3/examples/flags.md

```diff
@@ -126,6 +128,7 @@ func main() {
 	var count int
 
 	cmd := &cli.Command{
+		UseShortOptionHandling: true,
```

Comment:
There are no short options here, so this option looks useless.

Reply:
Hi @abitrolly. I didn't understand what you meant, are you saying that `UseShortOptionHandling` does not exist in v3? I just checked, it does exist. Ref: https://github.com/urfave/cli/blob/937cfe918cb10bd15aeeab4519ef2d6058a296ea/command.go#L108-L111

Reply:
@BlackHole1 I mean there are no short options in this example.

Reply:
I re-examined the modified context, and I confirm that this example does involve the `-vvv` scenario.

![21-12 11 21@2x](https://github.com/user-attachments/assets/31e27c31-7a71-4002-a8f6-e2018f5f811f)

Reply:
So how to use the short option to increase the count?

Reply:
I still don't understand the specific issue you're trying to address. Could you please describe it again?

The current changes in the PR are all in the `flags.md` of `v2` and `v3`, and the contextual descriptions have hardly changed. The only difference is that one uses `Flag.Count` and the other uses `Config.Count`, but the textual description is almost identical.
From your description, I get the impression that you mean: "The changes in v2 are fine, but there is a problem with the changes in v3."

The reason this PR changes here is that someone submitted an issue: https://github.com/urfave/cli/issues/1942, in which they pointed out that the current description of Flags in the documentation (which is where this PR is making changes) is misleading, causing them to be unable to run it.

Reply:
@BlackHole1 I think @abitrolly was saying that the `cli.BoolFlag` in the example doesn't have e.g. `Aliases: []string{"f"}` that gives the `foo` flag a short option.

https://github.com/urfave/cli/blob/937cfe918cb10bd15aeeab4519ef2d6058a296ea/docs/v2/examples/flags.md?plain=1#L127-L131

https://github.com/urfave/cli/blob/937cfe918cb10bd15aeeab4519ef2d6058a296ea/docs/v3/examples/flags.md?plain=1#L130-L136

I created an example here: https://go.dev/play/p/8nVATQqJz2F

Without `Aliases: []string{"f"}`, we would see
```
GLOBAL OPTIONS:
   --foo       foo greeting (default: false)
   --help, -h  show help (default: false)
```

With `Aliases: []string{"f"}`,
```
GLOBAL OPTIONS:
   --foo, -f   foo greeting (default: false)
   --help, -h  show help (default: false)
```

Reply:
> @BlackHole1 I think @abitrolly was saying that the `cli.BoolFlag` in the example doesn't have e.g. `Aliases: []string{"f"}` that gives the `foo` flag a short option.

This. @Juneezee thanks.

Reply:
Thanks! I understand.

Reply:
Fixed

Reply:
I added two suggestions that add short flags to parameters, so that the code is actually being used.